lib/workers/repository/process/fetch.ts

```diff
@@ -28,7 +28,8 @@ async function fetchDepUpdates(
   if (is.string(dep.depName)) {
     dep.depName = dep.depName.trim();
   }
-  if (!is.nonEmptyString(dep.depName)) {
+  dep.packageName ??= dep.depName;
+  if (!is.nonEmptyString(dep.packageName)) {
```

Comment:
The `depConfig as LookupUpdateConfig` on line 60 meant our type checking didn't catch/enforce this

Reply:
yes, there is a lot room for type improvement